src/internal_modules/roc_core/string_list.h

```diff
@@ -80,6 +80,11 @@ class StringList : public NonCopyable<> {
     //!  false if allocation failed.
     ROC_ATTR_NODISCARD bool push_back(const char* str);
 
+    //! Remove string from end of the list.
+    //! @returns
+    //!  false if deallocation failed.
+    ROC_ATTR_NODISCARD bool pop_back();
```

Comment:
Nitpick: please move method after push_back(), to keep two push_backs grouped together.

Reply:
Done